source/FreeRTOS_Routing.c

```diff
@@ -821,9 +821,9 @@ struct xIPv6_Couple
                            break;
                     #endif /* ( ipconfigUSE_IPv6 != 0 ) */
 
-                    #if ( ipconfigUSE_IPv4 != 0 )
-                        case ( BaseType_t ) pdFALSE:
-
+                    case ( BaseType_t ) pdFALSE:
```

Comment:
I think we decided not to do these kind of declaration. @tony-josi-aws to take a look.

Reply:
This is detected as DEADCODE by Coverity. xIsIPv6 is set to either pdTRUE or pdFALSE in the top of this function. So it's not possible to enter default case. And this is the solution for it.

Reply:
I think the only alternative is to take a MISRA exception here, but I believe this looks better than taking exception.